demo/src/app/components/+datepicker/datepicker-section.list.ts

```diff
@@ -173,6 +174,15 @@ export const demoComponentContent: ContentSection[] = [
           Sunday is considered the first day of the week and thus has the value 0</p>`,
         outlet: DemoDatepickerDaysDisabledComponent
       },
+      {
+        title: 'Dates disabled',
+        anchor: 'dates-disabled',
+        component: require('!!raw-loader?lang=typescript!./demos/disable-dates/disable-dates.ts'),
+        html: require('!!raw-loader?lang=markup!./demos/disable-dates/disable-dates.html'),
+        description: `
+          Decent description has to come here!!!`,
```

Comment:
@frederikprijck Could you, please add appropriate description here?

Reply:
I've updated the description.